READMEにデータベース設計を記載

fumi0204 opened the pull request:
What

1.READMEの拡張子を.mdに変更しました。
2.READMEにテーブル設計を記載しました。

Why
開発フローに必要のため

Commit:
READMEにデータベース設計を記載

What

1.READMEの拡張子を.mdに変更しました。
2.READMEにテーブル設計を記載しました。

Why
開発フローに必要のため

b-i-hide commented on README.md:
```diff
@@ -0,0 +1,48 @@
+| columm | type | not null | unique | index |
+|:-----------|------------:|:------------:|:------------:|:------------:|
+| name | string | true | true | true |
+| password | string | true | false | false |
```
これは必要ですか？？

b-i-hide commented on README.md:
```diff
@@ -0,0 +1,48 @@
+| name | string | true | true | true |
+| password | string | true | false | false |
+
+Association: has_many :groups, has_many : messages
```
group_userのアソシエーションも記述しましょう

b-i-hide commented on README.md:
```diff
@@ -0,0 +1,48 @@
+|:-----------|------------:|:------------:|:------------:|:------------:|
+| name | string | true | true | true |
+
+Association: has_many :users , has_many : messages
```
group_userのアソシエーションも記述しましょう。

Commit:
Usersテーブルのカラム修正、アソシエーションの追加

Usersテーブルのカラム修正(passwordを削除⇒emailの追加)と、UsersテーブルとGroupsテーブルにGroup_userテー
ブルのアソシエーションの追加をしました。

b-i-hide commented on README.md:
```diff
@@ -8,22 +8,24 @@
 |:-----------|------------:|:------------:|:------------:|:------------:|
 | name | string | true | true | true |
-| password | string | true | false | false |
+| email | string | true | false | false |
```
こちらも必要でしょうか？？

b-i-hide commented on README.md:
```diff
@@ -8,22 +8,24 @@
+| email | string | true | false | false |
 
-Association: has_many :groups, has_many : messages
+Association: has_many :groups, has_many : messages , has_many : group_user
```
この様にアソシエーションをくんでいただいているのですが、この組み方だとgroup_userを作る必要がなくなってしまいます。なのでアソシエーションの組み方を再度考えていただきたいです。

b-i-hide commented on README.md:
```diff
@@ -8,22 +8,24 @@
 | name | string | true | true | true |
 
-Association: has_many :users , has_many : messages
+Association: has_many :users , has_many : messages ,has_many : group_user
```
ここもですね。

Commit:
READMEの記述修正

修正点

1.Usersテーブルのemailカラムの削除
2.Usersテーブルのgroup_userとのアソシエーションの修正
3.Groupsテーブルのgroup_userとのアソシエーションの修正

b-i-hide commented on README.md:
```diff
@@ -15,17 +15,16 @@ Details of database design of "chat-space" are as follows.
-| email | string | true | false | false |
 
-Association: has_many :groups, has_many : messages , has_many : group_user
+Association: has_many: groups, has_many: messages , through: :group_user
```
しっかりと「has_many through: ~」と記述しましょう

fumi0204 replied:
修正done

b-i-hide commented on README.md:
```diff
@@ -15,17 +15,16 @@ Details of database design of "chat-space" are as follows.
 | name | string | true | true | true |
 
-Association: has_many :users , has_many : messages ,has_many : group_user
+Association: has_many: users , has_many: messages , through: : group_user
```
ここここもですね。

fumi0204 replied:
修正しました

b-i-hide commented:
お疲れ様です！
こちらがコメントしたものには何かしらの返答をいただけると見落としがないことの確認ができるので、そちらの方も次回からよろしくお願いいたします。
例）修正done、doneなど

Commit:
READMEの修正

has_many throughの修正
